test/kwtool_web/controllers/auth_controller_test.exs

```diff
@@ -1,10 +1,32 @@
 defmodule KwtoolWeb.AuthControllerTest do
   use KwtoolWeb.ConnCase, async: true
 
-  describe "sign_up/2" do
+  describe "get show/2" do
     test "returns 200 status", %{conn: conn} do
       conn = get(conn, "/sign_up")
-      assert html_response(conn, 200) =~ "Keyword Research Tool"
+
+      assert html_response(conn, 200) =~ "Registration"
+    end
+  end
+
+  describe "post create/2" do
+    test "renders errors when given valid registration params", %{conn: conn} do
+      password_inputs = %{password: "123456", password_confirmation: "123456"}
+
+      conn = post(conn, Routes.auth_path(conn, :create), user: params_for(:user, password_inputs))
```

Comment:
Since you already have request tests, maybe here, we can just call the controller directly

```elixir
describe "create/2" do
  ...
  AuthController.call(conn, :create, ...)
  ...
 end
```